src/components/WatershedSummaryTable/WatershedSummaryTable.js

```diff
@@ -120,7 +129,7 @@ export default class WatershedSummaryTable extends React.Component {
     }
 
     // Waiting for data
-    if (this.state.fetchingData || this.state.data === null) {
+    if (!this.state.data) {
```

Comment:
This appears to work great both at the outset and if a user clears a previously selected watershed outlet.